cmudb/env/README.md

```diff
@@ -2,16 +2,19 @@ The `env` folder is for all files that relate to setting up a development or CI
 
 # Docker tips.
 
-```
-# Build a Docker image named pgnp from our Dockerfile.
-docker build --tag pgnp --file ./cmudb/env/Dockerfile .
+1. Build the Dockerfile image `pgnp`: `sudo docker build --tag pgnp --file ./cmudb/env/Dockerfile .`
+2. Run either the:
+    - Single-node config: `sudo docker-compose -f cmudb/env/docker-compose-single.yml up`
+    - Replicated config: `sudo docker-compose -f cmudb/env/docker-compose-single.yml up`
```

Comment:
I think the path here should be `cmudb/env/docker-compose-replication.yml`

Reply:
good catch